src/main/java/com/github/manolo8/darkbot/modules/utils/LegacyFlashPatcher.java

```diff
@@ -81,18 +88,20 @@ private static class FlashInstaller implements Fixer {
         private static final Path
                 FLASH_DIR = Paths.get(System.getenv("APPDATA"), "DarkBot", "Flash"),
                 FLASH_OCX = FLASH_DIR.resolve("Flash.ocx"),
+                DARK_FLASH_OCX = Paths.get("", "lib", "DarkFlash.ocx"),
                 REG_SVR = Paths.get(System.getenv("WINDIR"), "SysWOW64", "regsvr32");
 
-
         public boolean needsFix() {
-            return !Files.exists(FLASH_DIR) || !Files.exists(FLASH_OCX);
+            return LibSetup.downloadLib("DarkFlash.ocx", FLASH_OCX);
         }
 
         public List<String> script() {
+            FileUtils.ensureDirectoryExists(FLASH_DIR);
```

Comment:
this may require admin perms for something in appdata, unsure tho

Reply:
no, because is appdata of the current user